README.md

```diff
@@ -32,6 +32,8 @@ Update deps
 mix deps.get
 ```
 
+To run `mix xcribe.doc` you must configure `preferred_cli_env: ["xcribe.doc": :test]` on your mix.ex file.
```

Comment:
One idea, in order to centralize the main documentation in one place, today we have main documentation and readme, with part of the text duplicated. 

Maybe we could use this technique https://github.com/thiamsantos/mxpanel/blob/main/lib/mxpanel.ex#L3 and just part of the readme as the main moduledoc.

Reply:
I think I will open a issue for it.